clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -270,6 +270,12 @@ default DeleteTopicsResult deleteTopics(TopicCollection topics) {
      */
     DeleteTopicsResult deleteTopics(TopicCollection topics, DeleteTopicsOptions options);
 
+    default CreateOrDeleteFederatedTopicZnodesResult deleteFederatedTopicZnodes(Map<String, String> xinfraTopics) {
```

Comment:
On both create/delete RPC, let's document that
1. This is decoupled from topic create/delete
2. This would eventually only change the ACL validation behavior within kafka-server level

Reply:
And by the way, `s/xinfra/federated/g`